packages/sanity/src/core/scheduledPublishing/tool/Tool.tsx

```diff
@@ -74,6 +76,19 @@ export default function Tool() {
     }
   }
 
+  if (!enabled) {
+    return (
+      <Container width={1} paddingTop={4}>
+        <Box paddingTop={4} paddingX={4}>
+          <ErrorCallout
+            description="You do not have permission to edit schedules."
```

Comment:
ℹ️  We don't have translations in `scheduledPublishing` 
Instead of showing the same error that we show when the request fails we add an specific not enabled error.
**Current error **
<img width="600" alt="Screenshot 2024-09-17 at 17 23 06" src="https://github.com/user-attachments/assets/03aba27d-1e56-4dbf-9c72-061c956381d8">

**Updated error**
<img width="600" alt="Screenshot 2024-09-17 at 17 24 34" src="https://github.com/user-attachments/assets/97743f88-a74e-4b3b-934a-6bbf9c6ad01e">